app/views/teaching_events/index.html.erb

```diff
@@ -6,26 +6,13 @@
       Find out more about getting into teaching at a free event where you can get all your questions answered.
     </p>
 
-    <details class="teaching-events__descriptions">
-      <summary class="teaching-events__descriptions--summary">Explore the different types of events</summary>
+    <h2 class="heading-s">Get Into Teaching events</h2>
```

Comment:
The alignment is a bit squiffy in this section now the accordion has been removed @philspray 

![image](https://github.com/DFE-Digital/get-into-teaching-app/assets/87642394/8bf92a1d-3638-4abd-960f-2f5153aed0e7)